index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Dynamic Landing Page</title>
+</head>
+<body>
+  <time id="time">00:00</time>
+    <h1>
+      <span id="greeting">HI!</span>
+      <span id="name" contenteditable="true">Mr. X</span>
+    </h1>
+
```

Comment:
In Html-file we usually don't add empty lines.

Reply:
DONE!